config/initializers/strong_migrations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# To mark migrations as safe that were created before installing this gem
+StrongMigrations.start_after = 20300101000000 # Disabled By default #2030/01/01
```

Comment:
?? with this you are disabling strong migrations till 2030, right? why would you do that?

Reply:
That configuration is there because we still don't know if we are going to leave it enabled initially or not. In the event that it is enabled, that should be removed

Reply:
with that config you are completely disabling the gem right? so what's the point of adding it in the first place?

Reply:
@brunoparma88 after the conversation we had I understand your point but I think we should have strong migrations enabled by default after the last existing migration

Reply:
@rootstrap/ruby thoughts?

Reply:
Agree with @santib. It could be nice to have it enabled for existing migrations though 🤔

Reply:
+1 to Santi and Seba's comments